Initial copy

ivanmvh opened the pull request:
I have a previously developed project related to a Product Landing Page, your approval is required to continue with the project. Basicly the core of the project is: 
- index.html

TIA
Ivan Martinez von Halle

ivanmvh committed:
Initial copy

ivanmvh commented on babel/index.babel:
```diff
@@ -0,0 +1,27 @@
+// !! IMPORTANT README:
+
+// You may add additional external JS and CSS as needed to complete the project, however the current external resource MUST remain in place for the tests to work. BABEL must also be left in place. 
+
+/***********
+INSTRUCTIONS:
+  - Select the project you would 
+    like to complete from the dropdown 
+    menu.
+  - Click the "RUN TESTS" button to
+    run the tests against the blank 
+    pen.
+  - Click the "TESTS" button to see 
+    the individual test cases. 
+    (should all be failing at first)
```
Because this a new project it's not neccesary the babel files but instead the instalation of linters in required. PLease do this ammends  are so important :ok_man:

ivanmvh replied:
ok you are right I will fix this.

ivanmvh commented on index.html:
```diff
@@ -0,0 +1,38 @@
+<header id="header">
+  <img src="https://cdnprod.mafretailproxy.com/sys-master-root/h95/ha8/14539669733406/404925_main.jpg_1700Wx1700H
+  "  id="header-img" width="100" height="200"/>
+  <nav id="nav-bar"> 
+    <button class="nav-link" type="button">Click Me!</button>
+    <button class="nav-link" type="button">Click Me!</button>
+    <button class="nav-link" type="button">Click Me!</button>
+  </nav>
```
The button must be in a grid container. Yours works fine now, but the requirement asks you to use a grid display because it's possible that in the future, when we adapt to desktop display, grid display will be important. :Cactus: . You're doing good, be patient.

ivanmvh replied:
I will fix this

victorokoroji replied:
[OPTIONAL]
- You did well add a nice review 👍🏼 . I think your review can be more polite because different people view suggestions differently. 

Instead of writing: `The button must be in a grid container`
You can write: `I think the button should be in a grid container` (I think this is more polite)

ivanmvh reviewed (commented):
That all for now, make the changes and ask a new Request Review. You are doing well :100: go ahead

victorokoroji reviewed (approved):
Hi @ivanmvh, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Good use of inline comments ✅ 
- Comments are polite ✅ 
- Comments include an explanation ✅ 

<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ivanmvh committed:
Delete of unnecesary folder and files Babel -js install Linters

- Delete of unnecesary folders and files
- Unnistall Babel
- delete js folder i
- install Linters

ivanmvh reviewed (commented):
You are right. All is ok ! :+1: :1st_place_medal:
